CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [Unreleased]
+
+#### Warning - Potentially breaking change
+- Restivus used to store the account login token in the user document: `services.resume.loginTokens.token`
```

Comment:
Sorry for my incorrect use of terminology. You're right, it's their token, not their credentials, that will be invalid. Thanks for catching that.